tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/StaticMethods.java

```diff
@@ -431,4 +438,79 @@ public static ObjectValue echoObject(ObjectValue obj) {
     public static boolean echoImmutableRecordField(MapValue value, BString key) {
         return value.getBooleanValue(key);
     }
+
+    public static Object acceptAndReturnReadOnly(Object value) {
```

Comment:
this method does not do anything related to readOnly right?

Reply:
This is the external implementation directly mapped to a Ballerina function that accepts and returns `readonly` typed values.

https://github.com/ballerina-platform/ballerina-lang/blob/0f66daefd505985b72d67c0967856844f05b7d01/tests/jballerina-unit-test/src/test/resources/test-src/javainterop/ballerina_types_as_interop_types.bal#L520

Reply:
ok